{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "46172ea8-9939-4dc1-b419-18d91343da54-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1604655339834, "stop": 1604655339849, "uuid": "22c2eb84-b375-4103-8c45-633095726484", "historyId": "babdba5e5940d3ba90e375f5ee53eb4e", "testCaseId": "f56f10c6b6679a14e1ccd1e8f399d334", "fullName": "test_userManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_userManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "19320-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_userManage"}]}